modules/cvc_check/root.sqf

```diff
@@ -2,4 +2,10 @@
 
 	#include "init.sqf"
 
-#endif
\ No newline at end of file
+#endif
+
+#ifdef description_functions
```

Comment:
New structure of function includes is 
```sqf
class CfgFunctions {
	class FW {
		tag = "FW";
		#define description_functions
		#include "core\description.hpp" //DO NOT REMOVE
		#include "modules\modules.sqf" //DO NOT REMOVE
		#undef description_functions
	};
	#define description_external_functions
	#include "core\description.hpp" //DO NOT REMOVE
	#include "modules\modules.sqf" //DO NOT REMOVE
	#undef description_external_functions
};
```

If you want functions to not have the FW tag prefix use `#ifdef description_external_functions` otherwise no general tag config for functions, aka 
```sqf
class cvc_check {
	tag = "kobld_cvc_check";
```
instead having just the function groups

Reply:
Fixed.